implants/eldritch/src/process/list_impl.rs

```diff
@@ -1,5 +1,109 @@
-use anyhow::Result;
+use anyhow::{Result};
+use sysinfo::{ProcessExt,System,SystemExt,PidExt,User,UserExt};
 
-pub fn list() -> Result<Vec<i32>> {
-    unimplemented!("Method unimplemented")
+pub struct ProcessRes {
+    pid:        u32,
+    ppid:       u32,
+    status:     String,
+    username:   String,
+    path:       String,
+    command:    String,
+    cwd:        String,
+    environ:    String,
+    name:       String,
+}
+
+impl ToString for ProcessRes {
```

Comment:
Rust recommends not implementing the ToString trait directly, but Display instead:
https://doc.rust-lang.org/stable/std/string/trait.ToString.html

Reply:
Good call!